src/hoc/withTimeseries.tsx

```diff
@@ -76,14 +77,17 @@ export const withTimeseries = <P extends WithTimeseriesDataProps>(
 
     async loadTimeseries() {
       try {
+        if (!this.context) {
```

Comment:
We with @mrtraser agreed that it's better to check whether the context exists only in `componentDidMount`

Reply:
Fixed